src/client/src/containers/ReviewAndGenerate/index.tsx

```diff
@@ -1,120 +1,82 @@
+    defaultMessage: "Launch Your Project"
   },
-  services: {
-    id: "review.services",
-    defaultMessage: "Services"
+  almostDone: {
+    id: "context.almostDone",
+    defaultMessage:
+      "You're almost done - review your project details on the right and make any necessary adjustments!"
   },
-  reviewAndGenerate: {
-    id: "review.reviewAndGenerate",
-    defaultMessage: "Your project summary."
+  giveFeedback: {
+    id: "about.reportAnIssueLabel",
+    defaultMessage: "Give Feedback"
   }
 });
 
 const ReviewAndGenerate = (props: Props) => {
-  const { servicesRows, intl, frameworkRows, projectName, outputPath } = props;
+  const { intl, openViewLicensesModal } = props;
+  const { formatMessage } = intl;
+
   return (
     <div className={styles.container}>
-      <Title>{intl.formatMessage(messages.reviewAndGenerate)}</Title>
-      <div className={styles.selectionContainer}>
-        <div className={styles.selectionTitle}>
-          {intl.formatMessage(messages.welcome)}
+      <div className={styles.reviewContextContainer}>
+        <div className={styles.selectionContainer}>
+          <Title>{formatMessage(messages.launchYourProject)}</Title>
+          {formatMessage(messages.almostDone)}
         </div>
-        <SummaryTile
-          showFolderIcon={true}
-          svgUrl={withLocalPath(folder)}
-          title={projectName}
-          subTitle={`${outputPath}/${projectName}`}
-        />
-      </div>
-      <SummarySection
-        selectionTitle={intl.formatMessage(messages.frameworks)}
-        selectionRows={frameworkRows}
-      />
-      <div className={styles.selectionContainer}>
-        <div className={styles.selectionTitle}>
-          {intl.formatMessage(messages.pages)}
+        <div className={styles.buttonContainer}>
+          <button
+            className={classnames(buttonStyles.buttonDark, styles.leftButton)}
+            onClick={openViewLicensesModal}
+          >
+            {formatMessage(messages.viewLicenses)}
+          </button>
+          <button
+            className={classnames(buttonStyles.buttonDark, styles.button)}
+          >
+            <a
+              className={styles.link}
+              href="https://github.com/Microsoft/WebTemplateStudio/issues"
+            >
+              {formatMessage(messages.giveFeedback)}
+            </a>
+          </button>
         </div>
-        <SortablePageList isSummaryPage={true} />
       </div>
-      <SummarySection
-        selectionTitle={intl.formatMessage(messages.services)}
-        selectionRows={servicesRows}
-        canDelete={true}
-      />
     </div>
   );
 };
 
-const mapDispatchToProps = (
-  dispatch: ThunkDispatch<AppState, void, RootAction>
-): IDispatchProps => ({
-  openCosmosDbModal: () => {
-    dispatch(ModalActions.openCosmosDbModalAction());
-  },
-  openAzureFunctionsModal: () => {
-    dispatch(ModalActions.openAzureFunctionsModalAction());
+const mapDispatchToProps = (dispatch: any): IDispatchProps => ({
```

Comment:
Why did you change the type of dispatch to `any`?

Reply:
Good catch, I was just following another Modal's code and avoid unnecessary libraries, but it will be better to use "dispatch: Dispatch<ModalActionType>". Will update my PR.